File: app/src/main/java/ir/example/model/LessonModel.java
package ir.example.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;

@RealmClass(name = "Lesson")
public class LessonModel extends RealmObject {
    @PrimaryKey
    public long id;
    public String name;
}

File: app/src/main/java/ir/example/model/LessonModel.kt
package ir.example.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass

@RealmClass(name = "Lesson")
open class LessonModel: RealmObject() {

    @PrimaryKey
    var id:Long =0
    var name:String =""

}